docs/index.rst

```diff
@@ -12,15 +12,13 @@ Next Steps
 
 `Getting started <getting_started.html>`_
 
-`Tutorials <tutorials/index.html>`_
-
 .. toctree::
     :hidden:
 
     Overview <self>
     Getting Started <getting_started>
-    Tutorials <tutorials/index>
     API Reference <apidocs/qiskit_nature_pyscf>
+    README <https://github.com/qiskit-community/qiskit-nature-pyscf/blob/main/README.md>
```

Comment:
Rather than this I was actually thinking to add this link to the text above.
Something like the following:

> **Qiskit Nature PySCF** is a third-party plugin of Qiskit Nature + PySCF.
> To get started we suggest looking at the `README on Github <https://github.com/qiskit-community/qiskit-nature-pyscf/blob/main/README.md>`_.